src/job/stagingJobHandler.ts

```diff
@@ -49,7 +49,7 @@ export class StagingJobHandler extends JobHandler {
   prepDeployCommands(): void {
     this.currJob.deployCommands = [
       `cd repos/${getDirectory(this.currJob)}`,
-      `make next-gen-stage${this.currJob.payload.pathPrefix ? `MUT_PREFIX=${this.currJob.payload.mutPrefix}` : ''}`,
+      `make next-gen-stage${this.currJob.payload.pathPrefix ? ` MUT_PREFIX=${this.currJob.payload.mutPrefix}` : ''}`,
```

Comment:
NIT: I think we can remove this white space

Reply:
I'd prefer to keep it, if that's ok! I'll explain:

There is a test for make next-gen-stage that expecets the whitespace there for when `this.currJob.payload.pathPrefix` evaluates to true here. The reason it wasn't failing before is that this.currJob.payload.pathPrefix, AFAIK, always evaluates to a falsy value at this point in the code. 
This is because getPathPrefix() is called in jobHandler.ts and sets the payload's pathPrefix, but for staging jobs, it always returns an empty string. I realized the test would fail if there was no extra whitespace because when I was testing how the prefix works with some staging jobs, I overrode how pathPrefix is currently set so that it would always evaluate to a truthy value.